guide/onboarding/creating-a-new-wallet.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: guide
+title: Creating a new wallet
+description: Steps to take when creating a new wallet
+parent: Onboarding
+nav_order: 2
+permalink: /guide/onboarding/creating-a-new-wallet/
+main_classes: -no-top-padding
+---
+
+# Creating a new wallet
+
+First, bitcoin [wallets]({{ 'guide/glossary/#wallet' | relative_url }}) are not analogous to their namesakes. Physical wallets are places to store physical currency, credit cards, IDs, and so on. Bitcoin wallets don't store bitcoin.
+
+Instead, a bitcoin wallet contains the "private keys" to your bitcoin. The bitcoin is located on a network that contains a public ledger similar in many ways to a spreadsheet with a record of everyone's transactions and balances (so think of bitcoin wallets more like keychains).
```

Comment:
Thanks for the suggestions.
 - The sentence before we say the bitcoin is not stored in a wallet and your change says the reverse straight after. I don't think it's accurate to say `bitcoin wallets store bitcoin` 
 -  Agree that they store other forms of data, but that is not their primary mechanism from my understanding they are primarily a key generation and used for signing transactions. I'm also making this comparison to what someone might be used to in terms of a physical wallet.
 - Tried to avoid the term blockchain here are just describe what it does by using something a think is more relatable e.g spreadsheet 
 - Agree, will update to make sure it's clear that it is the Bitcoin network.
 - Agree, nice suggestion.
 - Agree, another nice suggestion.

Updated to use a few of the suggested changes.

Reply:
> The sentence before we say the bitcoin is not stored in a wallet and your change says the reverse straight after. I don't think it's accurate to say bitcoin wallets store bitcoin

I changed it to `stores bitcoin private keys` - It's the same as the original sentence just more concise. 

Glad I could help out!